subprojects/docs/src/docs/userguide/gradle_wrapper.adoc

```diff
@@ -156,13 +156,6 @@ $ ./gradlew wrapper --gradle-version {gradleVersion}
 include::{samplesPath}/userguide/wrapper/wrapperGradleVersionUpgrade.out[]
 ----
 
-.Checking the Wrapper version after upgrading
-[listing,subs=+attributes]
-----
-$ ./gradlew -v
-include::{samplesPath}/userguide/wrapper/wrapperGradleVersionAfterUpgrade.out[]
```

Comment:
I think you can remove this file if it is not references elsewhere.

Reply:
Right. Missed that. Thanks.

Reply:
@britter Would you mind taking another look?